api/worker/api.go

```diff
@@ -2,17 +2,26 @@ package worker
 
 import (
 	"github.com/gorilla/mux"
+	"github.com/ufcg-lsd/arrebol-pb/arrebol/worker/auth"
+	"github.com/ufcg-lsd/arrebol-pb/arrebol/worker/manager"
+	"github.com/ufcg-lsd/arrebol-pb/storage"
 	"log"
 	"net/http"
 )
 
 type WorkerApi struct {
```

Comment:
there are many directories, like auth, token, whitelist that give me the feeling they should live in this directory, because they deal with things realated to the worker api. What do you think?

Reply:
for example, the manager manages the worker's operations, like join...

Reply:
Well, I'm following the pattern where / api keeps only the definition of routes and the implementation of its handlers. I moved the token and whitelist into the auth package because they are part of authentication domain.